blazar/manager/enforcement.py

```diff
@@ -53,7 +53,10 @@
     cfg.FloatOpt('usage_default_allocated',
                  default=20000.0,
                  help='Default allocation if project is missing from usage '
-                      'DB.')
+                      'DB.'),
+    cfg.StrOpt('project_enforcement_id',
+                 help='Unique identifier to use for projects. '
```

Comment:
I would put `default='name'` here if it is really a default. I see there is handling for the case when the attribute lookup fails; I think that's fine to keep in there in case they do something like `project_enforcement_id=missingkey`